### ISSUE TITLE
Potential new B.1.1.28 sub-lineage with L452R in Brazil

### ISSUE THREAD

**Issue by cibittar** [cov-lineages/pango-designation#68]
by Cíntia Bittar, Fábio Sossai Possebon; Leila Sabrina Ullmann; Luiz G P de Almeida; Cecília Artico Banho; Guilherme Campos; Helena Lage Ferreira; Jorge A. Petroli Marchesi; Lívia Sacchetto; Maisa C. Pereira Parra; Marília Moraes; Ana Tereza R Vasconcelos; Paulo Inácio da Costa; Maurício L. Nogueira; Paula Rahal; João Pessoa Araújo Jr.

### **Description**

Sub-lineage of: B.1.1.28
Earliest sequence: 2021-02-18
Most recent sequence: 2021-04-05
Countries circulating: Brazil

Here we present a potential new sub-lineage of B.1.1.28 harboring L452R mutation of Spike protein.  Sequences from cities from the Araraquara – São Paulo – Brazil region, classified by PANGO as B.1.1.28, presented a set of mutations including the L452R in Spike protein. We assembled a dataset of 337 sequences including GISAID Refseq WIV04, B.1.1.28 (297), P.1 (11), P.2 (7), P.3 (3) and all available sequences of B.1.1.28 (19) with the new mutation. The maximum likelihood phylogenetic tree presented below (Figure 1) shows that the sequences group in a monophyletic branch with strong branch support (aLRT). We present a map of Brazil, highlighting the state of São Paulo and the cities where the samples were collected (Figure 2). In Figure 1C, tree tip labels were replaced by the city where samples were collected. 
We also analyzed the set of mutations shared by the sequences that compose the clade. There are 13 non-synonymous mutations and 5 synonymous mutations that are not present together in other B.1.1.28 sequences (Table 1). There are two exceptions as sequence EPI_ISL_1580269 does not present C20436T and C22079A / S : Q173K and EPI_ISL_1625982 does not present A23720G / S : I720V mutation. Since these are the two outer sequences in the clade, this might be an indication of the epidemiological path of this potential new sub-lineage.

### **Genomes**

Here are the IDs of the genomes

hCoV-19/Brazil/SP-2223/2021\|EPI_ISL_1520132\|2021-02-19
hCoV-19/Brazil/SP-2224/2021\|EPI_ISL_1520133\|2021-02-19
hCoV-19/Brazil/SP-48379-21/2021\|EPI_ISL_1580269\|2021-03-31
hCoV-19/Brazil/SP-2281/2021\|EPI_ISL_1625982\|2021-02-17
hCoV-19/Brazil/SP-2282/2021\|EPI_ISL_1625983\|2021-02-18
hCoV-19/Brazil/SP-BT18005/2021\|EPI_ISL_1734841\|2021-03-24
hCoV-19/Brazil/SP-BT19806/2021\|EPI_ISL_1734875\|2021-04-05
hCoV-19/Brazil/SP-827819/2021\|EPI_ISL_1795084\|2021-03-21
hCoV-19/Brazil/SP-827826/2021\|EPI_ISL_1795085\|2021-03-21
hCoV-19/Brazil/SP-827926/2021\|EPI_ISL_1795086\|2021-03-21
hCoV-19/Brazil/SP-833757/2021\|EPI_ISL_1795087\|2021-03-22
hCoV-19/Brazil/SP-833783/2021\|EPI_ISL_1795088\|2021-03-22
hCoV-19/Brazil/SP-833797/2021\|EPI_ISL_1795089\|2021-03-22
hCoV-19/Brazil/SP-833805/2021\|EPI_ISL_1795090\|2021-03-22
hCoV-19/Brazil/SP-827827/2021\|EPI_ISL_1795091\|2021-03-21
hCoV-19/Brazil/SP-833817/2021\|EPI_ISL_1795092\|2021-03-21
hCoV-19/Brazil/SP-26656/2021\|EPI_ISL_1795401\|2021-03-24
hCoV-19/Brazil/SP-827543/2021\|EPI_ISL_1795412\|2021-03-21
hCoV-19/Brazil/SP-827835/2021\|EPI_ISL_1795413\|2021-03-21

### **Evidence**

**Figure 1.** A. Maximum likelihood phylogenetic tree. B. Detail of the branch of the potential new sub-lineage. C. Detail of the branch of the potential new sub-lineage with tip labels replaced by the city where the samples were collected.

![Figure1_PANGO](https://user-images.githubusercontent.com/20564408/117148304-7ed75500-ad8c-11eb-8524-ba2ba93961b0.jpg)

**Figure 2.** Map of Brazil highlighting the State of São Paulo (gray) and the cities where the samples were collected (red).

![Figure2_PANGO](https://user-images.githubusercontent.com/20564408/117148440-98789c80-ad8c-11eb-853d-5785bc671124.jpg)

**Table 1.** Clade defining mutations.

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List
href="file:///C:\Users\Genoma\AppData\Local\Temp\msohtmlclip1\01\clip_filelist.xml">

<link rel=themeData
href="file:///C:\Users\Genoma\AppData\Local\Temp\msohtmlclip1\01\clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:\Users\Genoma\AppData\Local\Temp\msohtmlclip1\01\clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=PT-BR style='tab-interval:35.4pt'>

<div align=center>

Genomic region | Nucleotide | Amino acid
-- | -- | --
  | A136G | -
ORF1a | G1811A | A516T
ORF1a | C3177T | P971L
ORF1a | C9693T | A3143V
ORF1a | T9867C | L3201P
ORF1a | C11450A | Q3729K
ORF1a | C12008T | L3915F
ORF1a | C12880T | -
ORF1b | A15932G | Y822C
ORF1b | G18756T | -
ORF1b | C20436T* | -
S | G21987T | G142V
S | C22079A* | Q173K
S | T22917G | L452R
S | A23720G** | I720V
ORF3a | G25540A | V50I
  | A28271C | -
N | C28932T | A220V

</div>

</body>

</html>

**Comment by cibittar** [comment]
Dear @chrisruis,
We have been observing an increase in the circulation of this new lineage in the city of Porto Ferreira – SP, which is the city we receive samples for genomic surveillance. The first detection of the new lineage in the city happened in a sample from March 2021, when P.1 lineage was already the most frequent. P.1 is a very aggressive Variant of Concern (VOC) and rapidly replaces most of the other lineages after it has been introduced into a new city in Brazil (1). We have been tracking lineages through Sanger sequencing of the RBD region of the Spike protein and the data show the new sub-lineage emerged in a moment P.1 was already the most prevalent and is increasing in frequency ever since, suggesting it can compete with P.1. Figure 1 shows the distribution of the lineages in February (n=5), March (n=18), April (n=67) and May (n=46) where we can see it is increasing its frequency. 
Since the last post we have deposited in GISAID new sequences of the new lineage from Porto Ferreira – SP and Descalvado – SP and other groups also added sequences from other cities from the State. The new Maximum Likelihood Phylogenetic tree, including all sequences of B.1.1.28 with L452R available in GISAID, is shown below. Again, all sequences with the characteristics previously described, grouped into a monophyletic branch with strong support (Figure 2 and 3).  The new lineage is now circulating in 21 cities of the State of São Paulo as can be seen in Figure 4 and 5. Table 1 displays all the information of the sequences from the new lineage deposited in GISAID. 
We kindly ask if you can take another look into this new lineage. The presence of the L452R mutation in this clade is very concerning due to the evidences that L452R reduces antibody neutralization and the impact that other lineages that bear these mutations are causing in the world (B.1.427/B.1.429 – California and B.1.627 – India) (2). We believe we have detected this new lineage in an early stage of circulation, but as shown above, it has been increasing in frequency and spreading fast, even in the presence of P.1. Actions could be taken at the moment to prevent its spread to other regions of Brazil and also to other countries in the world, as has happened to P.1 and B.1.627. Even though local and federal authorities were informed, until it is designated as a new lineage no extra measures will be taken to contain its circulation. 

![Figure 1](https://user-images.githubusercontent.com/20564408/119243713-c6f1c800-bb3f-11eb-9d26-b3e7b43a3125.jpg)
**Figure 1.** Distribution of SARS-CoV lineages in Porto Ferreira – SP from February to May 2021.

![Figure 2](https://user-images.githubusercontent.com/20564408/119243721-d83ad480-bb3f-11eb-8101-1b796d049efa.JPG)
**Figure 2.** Maximum likelihood phylogenetic tree based on a dataset with 405 complete genome sequences. The red arrow highlights the new lineage.

![Figure 3](https://user-images.githubusercontent.com/20564408/119243736-f6083980-bb3f-11eb-9a5e-a03fcb0d53ef.JPG)
**Figure 3.** Detail of the branch of the new lineage, showing branch support.

![Figure 4](https://user-images.githubusercontent.com/20564408/119243744-0fa98100-bb40-11eb-8196-4182021dd3ef.JPG)
**Figure 4.** Detail of the branch of the new lineage with tip labels replaced by the city where the samples were collected.

![Figure 5](https://user-images.githubusercontent.com/20564408/119243752-1cc67000-bb40-11eb-9460-311c0c3297a6.jpg)
**Figure 5.** Map of the State of São Paulo – Brazil, highlighting the cities where the new lineage was sampled.



**Table 1.** Information of the sequences from the new lineage deposited in GISAID.





<meta name="ProgId" content="Word.Document">
<meta name="Generator" content="Microsoft Word 14">
<meta name="Originator" content="Microsoft Word 14">
<link rel="File-List" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_filelist.xml">

<link rel="themeData" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_themedata.thmx">
<link rel="colorSchemeMapping" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_colorschememapping.xml">

<style>

</style>







GISAID   ID | City | Sampling   Date | Submitting   Lab
-- | -- | -- | --
EPI_ISL_1520132 | Mococa | 2021-02-19 | Instituto   Adolfo Lutz
EPI_ISL_1520133 | Mococa | 2021-02-19 | Instituto   Adolfo Lutz
EPI_ISL_1580269 | Cesário   Lange | 2021-03-31 | Rede de   Vigilância Genômica (Vigenômica)/UNESP
EPI_ISL_1625982 | Itirapina | 2021-02-17 | Instituto   Adolfo Lutz
EPI_ISL_1625983 | Araras | 2021-02-18 | Instituto   Adolfo Lutz
EPI_ISL_1734841 | Porto   Ferreira | 2021-03-24 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_1734875 | Porto   Ferreira | 2021-04-05 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_1795084 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795085 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795086 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795087 | Santa   Cruz Das Palmeiras | 2021-03-22 | Instituto   Butantan
EPI_ISL_1795088 | Santa   Cruz Das Palmeiras | 2021-03-22 | Instituto   Butantan
EPI_ISL_1795089 | Santa   Cruz Das Palmeiras | 2021-03-22 | Instituto   Butantan
EPI_ISL_1795090 | Santa   Cruz Das Palmeiras | 2021-03-22 | Instituto   Butantan
EPI_ISL_1795091 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795092 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795401 | Rio   Claro | 2021-03-24 | Instituto   Butantan
EPI_ISL_1795412 | Sumaré | 2021-03-21 | Instituto   Butantan
EPI_ISL_1795413 | Tambaú | 2021-03-21 | Instituto   Butantan
EPI_ISL_1911963 | Porto   Ferreira | 2021-04-03 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_1966238 | Caconde | 2021-03-16 | Instituto   Butantan / Mendelics
EPI_ISL_1966541 | Santa   Cruz Das Palmeiras | 2021-03-29 | Instituto   Butantan / FZEA-USP (Pirassununga)
EPI_ISL_1966554 | Tapiratiba | 2021-03-29 | Instituto   Butantan / FZEA-USP (Pirassununga)
EPI_ISL_1966770 | Itapetininga | 2021-04-14 | Instituto   Butantan / Mendelics
EPI_ISL_1966858 | Capão   Bonito | 2021-04-08 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1966917 | Iperó | 2021-04-08 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1966943 | Itapira | 2021-04-05 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1966944 | Itapira | 2021-04-05 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1966948 | Itapira | 2021-04-05 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1967060 | São José   do Rio Pardo | 2021-04-05 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_1967061 | São   Miguel Arcanjo | 2021-04-08 | Instituto   Butantan / ESALQ-USP (Piracicaba)
EPI_ISL_2170978 | Porto   Ferreira | 2021-04-19 | Instituto   Butantan / Mendelics
EPI_ISL_2223407 | Porto   Ferreira | 2021-04-01 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223410 | Porto   Ferreira | 2021-04-01 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223414 | Porto   Ferreira | 2021-04-05 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223416 | Porto   Ferreira | 2021-04-12 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223419 | Porto   Ferreira | 2021-04-14 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223425 | Porto   Ferreira | 2021-04-16 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223427 | Porto   Ferreira | 2021-04-16 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223430 | Porto   Ferreira | 2021-04-20 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223431 | Porto   Ferreira | 2021-04-22 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223433 | Porto   Ferreira | 2021-04-22 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223435 | Porto   Ferreira | 2021-04-22 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223422 | Descalvado | 2021-04-16 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2223438 | Descalvado | 2021-04-25 | Instituto   de Biotecnologia - UNESP-Botucatu-SP
EPI_ISL_2209288 | Itapira | 2021-04-12 | Instituto   Butantan
EPI_ISL_2209353 | Rio   Claro | 2021-03-31 | Instituto   Butantan
EPI_ISL_2209379 | Rio   Claro | 2021-03-24 | Instituto   Butantan
EPI_ISL_2209593 | Cordeirópolis | 2021-03-25 | Instituto   Butantan
EPI_ISL_2209600 | Cordeirópolis | 2021-03-31 | Instituto   Butantan
EPI_ISL_2209732 | Ipeúna | 2021-04-12 | Instituto   Butantan
EPI_ISL_2209739 | Ipeúna | 2021-04-15 | Instituto   Butantan
EPI_ISL_2209746 | Ipeúna | 2021-04-15 | Instituto   Butantan
EPI_ISL_2209872 | Araras | 2021-04-12 | Instituto   Butantan
EPI_ISL_2209888 | Araras | 2021-04-14 | Instituto   Butantan
EPI_ISL_2210253 | Pirassununga | 2021-04-12 | Instituto   Butantan
EPI_ISL_2210275 | Pirassununga | 2021-04-13 | Instituto   Butantan
EPI_ISL_2210279 | Pirassununga | 2021-04-14 | Instituto   Butantan





1.	Genomic Surveillance of SARS-CoV-2 in the State of Rio de Janeiro, Brazil: technical briefing - SARS-CoV-2 coronavirus / nCoV-2019 Genomic Epidemiology. Virological https://virological.org/t/genomic-surveillance-of-sars-cov-2-in-the-state-of-rio-de-janeiro-brazil-technical-briefing/683 (2021).
2.	Li, Q. et al. The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity. Cell 182, 1284-1294.e9 (2020).

**Comment by chrisruis** [comment]
Thanks for all that extra information @cibittar Based on all that we've added this clade as lineage P.4 in [v1.2.9](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.9)

**Comment by cibittar** [comment]
Dear @chrisruis 
We have noticed that many sequences are being classified by Pango as P.4, following the update v1.29, but they are not actually P.4, since they do not present the same mutations. This is causing a lot of confusion for epidemilogical tracking of this lineage. Can you check this issue,  please?

**Comment by chrisruis** [comment]
Hi @cibittar definitely, I'll take a look. I'll have a look at everything assigned to P.4 but do you have IDs of any of the sequences that shouldn't be P.4 just to make sure I include those?

**Comment by cibittar** [comment]
Hi @chrisruis,
Here are some of our sequences (excel file) that up until yesterday were being classified by Pango as P.4 but are actually P.1. I have noticed that you have reclassified some of the sequences, since yesterday GISAID was returning more than 3000 results for P.4 and today its 1,490. However, it is still considering many sequences that are not P.4. I am also sending the mutations that define the P.4 clade below. Of course, some sequences may lack some of this mutations. A good way to track the real ones is by filtering P.4 and Spike_L452R mutation, which today returns 117 results. I hope this helps!

Best






<meta name="ProgId" content="Word.Document">
<meta name="Generator" content="Microsoft Word 14">
<meta name="Originator" content="Microsoft Word 14">
<link rel="File-List" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_filelist.xml">

<link rel="themeData" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_themedata.thmx">
<link rel="colorSchemeMapping" href="file:///C:\Users\cibittar\AppData\Local\Temp\msohtmlclip1\01\clip_colorschememapping.xml">

<style>

</style>






<div align="center">


Genomic region | Nucleotide | Amino acid
-- | -- | --
&nbsp; | A136G | -
ORF1a | G1811A | A516T
ORF1a | C3177T | P971L
ORF1a | C9693T | A3143V
ORF1a | T9867C | L3201P
ORF1a | C11450A | Q3729K
ORF1a | C12008T | L3915F
ORF1a | C12880T | -
ORF1b | A15932G | Y822C
ORF1b | G18756T | -
ORF1b | C20436T | -
S | G21987T | G142V
S | T22054G | N164K
S | C22079A | Q173K
S | T22917G | L452R
S | C23673T | S704L
S | A23720G | I720V
ORF3a | G25540A | V50I
ORF8 | C28253T | -
&nbsp; | A28271C | -
N | C28932T | A220V



</div>






[Misclassified P.1 sequences.xlsx](https://github.com/cov-lineages/pango-designation/files/6670207/Misclassified.P.1.sequences.xlsx)

**Comment by chrisruis** [comment]
Thanks @cibittar it looks like with today's pangolin, your misclassified sequences are now being assigned to P.1 which is great! Let me know if that changes though.

I've looked through all of the sequences currently _assigned_ to P.4 on GISAID which has reduced again to only 217 today. However, quite a few of those still aren't P.4. I think there's 2 things we can do to try and fix this. I've added 70 new designations to P.4 which definitely cluster in there and have L452R which will hopefully help. They've been added to [v1.2.15](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.15). The second thing is to figure out which lineages the sequences incorrectly designated to P.4 actually belong to and add them to the designations as those lineages. I'm just getting everything together for that now but it might take a couple of days to finish up. I'll let you know once that's done.
